src/Http/RequestMatcher/FosRequestMatcher.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCacheBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCacheBundle\Http\RequestMatcher;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestMatcher as SymfonyRequestMatcher;
+
+class FosRequestMatcher extends SymfonyRequestMatcher
```

Comment:
can you add a docblock saying that this extends the symfony request matcher to add query string matching?